src/qt/guiutil.cpp

```diff
@@ -40,11 +40,13 @@
 #include <QFontDatabase>
 #include <QFontMetrics>
 #include <QGuiApplication>
+#include <QJsonDocument>
```

Comment:
does this really need JSON document? we use it nowhere else, and is not really our log format anway

Reply:
I guess we could just print `QJsonObject QStaticPlugin::rawMetaData()`.

Reply:
> I guess we could just print `QJsonObject QStaticPlugin::rawMetaData()`.

It prints something like that:
```
2020-11-15T18:02:40Z [main] Static plugins:
2020-11-15T18:02:40Z [main]  QTMETADATA  qbjs\x01
```

Reply:
@laanwj @promag Thanks! [Updated](https://github.com/bitcoin-core/gui/pull/72#issuecomment-727617533).